pkgs/development/compilers/tinygo/default.nix

```diff
@@ -1,64 +1,74 @@
-{ lib, buildGoModule, fetchFromGitHub, llvm, clang-unwrapped, lld, avrgcc
-, avrdude, openocd, gcc-arm-embedded, makeWrapper, fetchurl }:
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, makeWrapper
+, llvm
+, clang-unwrapped
+, lld
+, avrgcc
+, go
+, gcc-arm-embedded
```

Comment:
Note that recent versions of TinyGo no longer need a GCC/binutils toolchain. Everything is included already: clang as C compiler, lld as linker, and picolibc/compiler-rt as libc.
(Eventually, it'll hopefully also drop the avrgcc dependency but that's a bit more complicated.)